exercises/practice/rna-transcription/zcl_rna_transcription.clas.testclasses.abap

```diff
@@ -0,0 +1,60 @@
+*"* use this source file for your ABAP unit test classes
+CLASS ltcl_rna_transcription DEFINITION FINAL FOR TESTING
+  DURATION SHORT
+  RISK LEVEL HARMLESS.
+
+  PRIVATE SECTION.
+    DATA cut TYPE REF TO zcl_rna_transcription.
+    METHODS setup.
+    METHODS:
+      empty_rna_sequence                    FOR TESTING,
+      rna_complement_of_cytosine_is_guanine FOR TESTING,
+      rna_complement_of_guanine_is_cytosine FOR TESTING,
+      rna_complement_of_thymine_is_adenine  FOR TESTING,
+      rna_complement_of_adenine_is_uracil   FOR TESTING,
```

Comment:
shorten the name of these, they are longer than 30 characters

Reply:
I have shortened the unit test names, and this has resolved this issue. Thank you.